### PR TITLE
Updated requirements file to remove 'make' entry

### PR DESCRIPTION
**Pull request by sayee-jadhav**
SUMMARY
A recent PR was created to add an entry to 'make'. However, it seems to be a /usr/bin/make, and the make entry might be installing the wrong "make". Therefore, removed the entry 'make' from the doc.

ISSUE TYPE
Docs Pull Request

COMPONENT NAME
docs.ansible.com

ADDITIONAL INFORMATION
There's no standard way but a script could be written to install 'make' specifically for rhel (edited) 
(on rhel: sudo yum install -y make). Note that unlike the other libraries in requirements.txt, make had to be installed by someone who can become a higher privileged user. (Info. shared by @abadger ). 
Earlier PR: https://github.com/ansible/ansible/pull/70020

### CODE REVIEW

**Comment by abadger** [comment]
+1 from me!  Thanks :-)

File: docs/docsite/rst/dev_guide/developing_locally.rst [hunk 1]
```diff
@@ -5,41 +5,45 @@
...
-To save a local module or plugin so Ansible can find and use it, drop the module or plugin in the correct "magic" directory. For local modules, use the name of the file as the module name: for example, if the module file is ``~/.ansible/plugins/modules/local_users.py``, use ``local_users`` as the module name.
+To save a local module or plugin such that Ansible can find and use it, add the module or plugin in the appropriate directory (the directories are specified in later parts of this topic). 
+
+.. contents::
+   :local:
 
 .. _modules_vs_plugins:
 
-Modules and plugins: what's the difference?
+What are modules and plugins?
```
Comment (Akasurde) [inline docs/docsite/rst/dev_guide/developing_locally.rst]:
```suggestion
What are modules and plugins?
=============================
```

File: docs/docsite/rst/dev_guide/developing_locally.rst [hunk 2]
```diff
@@ -5,41 +5,45 @@
...
 
-The easiest, quickest, and most popular way to extend Ansible is to copy or write a module or a plugin for local use. You can store local modules and plugins on your Ansible control node for use within your team or organization. You can also share a local plugin or module by embedding it in a role and publishing it on Ansible Galaxy. If you've been using roles off Galaxy, you may have been using local modules and plugins without even realizing it. If you're using a local module or plugin that already exists, this page is all you need.
+If you are using an existing module or plugin but Ansible can’t find it, this page is all you need. However, if you want to create a plugin or a module, go to :ref:`Developing plugins` and :ref:`Ansible module development: getting started` topics and then return to this page to know how to add it locally.
```
Comment (Akasurde) [inline docs/docsite/rst/dev_guide/developing_locally.rst]:
```suggestion
If you are using an existing module or plugin but Ansible can not find it, this page is all you need. However, if you want to create a plugin or a module, go to :ref:`Developing plugins` and :ref:`Ansible module development: getting started` topics and then return to this page to know how to add it locally. 
```

**Comment by Akasurde** [comment]
@sayee-jadhav Can you please rebase this? Thanks.

File: docs/docsite/rst/dev_guide/developing_locally.rst (hunk 1 above)
Comment (sayee-jadhav) [reply docs/docsite/rst/dev_guide/developing_locally.rst]: Thank you for the review @Akasurde. I edited the content and committed the changes in PR https://github.com/ansible/ansible/pull/70659.

File: docs/docsite/rst/dev_guide/developing_locally.rst (hunk 2 above)
Comment (sayee-jadhav) [reply docs/docsite/rst/dev_guide/developing_locally.rst]: Thank you for the review @Akasurde. I edited the content and committed the changes in PR https://github.com/ansible/ansible/pull/70659.

**Comment by sayee-jadhav** [comment]
Thank you for the review @Akasurde. I edited the content and committed the changes in PR https://github.com/ansible/ansible/pull/70659.

**Pull request closed without merging**